uint8 ALLOCATE = 0
uint8 CLASSIFY = 1
uint8 RESET = 2
uint8 DELETE = 3

uint8 action

# allocate/classify/reset/delete
string net_id

# allocate
string model_path

# classify
float64[] inputs

---
# results
uint8 OK                   = 0
uint8 ID_OCCUPIED          = 1 #allocate
uint8 MODEL_NOT_FOUND      = 2 #allocate
uint8 INVALID_MODEL        = 3 #allocate
uint8 NET_NOT_ALLOCATED    = 4 #classify/reset/delete
uint8 INCORRECT_INPUT_SIZE = 5 #classify
uint8 ACTION_UNKNOWN       = 6

uint8 result

# allocate result
string net_id

# classify result
uint8 c
float64 score
